lib/napa/cli/generate.rb

```diff
@@ -13,6 +14,13 @@ class Generate < Thor
         'Create a Database Migration'
       )
 
+      register(
+        Model,
+        'model',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.